module5/js/script.js

```diff
@@ -103,6 +103,8 @@ function buildAndShowHomeHTML (categories) {
       // variable's name implies it expects.
       // var chosenCategoryShortName = ....
 
+      var chosenCategoryShortName = chooseRandomCategory(categories);
```

Comment:
this step is incorrect. Pay attention to the comment: "Pay attention to what type of data that function returns vs what the chosenCategoryShortName variable's name implies it expects."
you can console.log to print out what `chooseRandomCategory(categories)` returns and compare with what is required here